Client/Cliqz/Frontend/Browser/ControlCenter/AntitrackingPanel.swift

```diff
@@ -559,6 +564,15 @@ class AntitrackingPanel: ControlCenterPanel {
         button.addSubview(separatorView2)
         
         
+        if let privateMode = self.isPrivateMode{
```

Comment:
better to use `if let privateMode = self.isPrivateMode where privateMode == true {`